tcms/testruns/tests/test_views.py

```diff
@@ -454,6 +456,18 @@ def test_update_selected_case_runs(self):
         self.assertContains(response, _('%d CaseRun(s) updated:') % 1)
         self.assertEqual(self.case_run_1.case.text, "Scenario Version 2")
 
+    def test_update_selected_case_runs_without_permissions(self):
+        remove_perm_from_user(self.tester, 'testruns.change_testcaserun')
+
+        self.assertNotEqual(self.case_run_1.case.history.latest().history_id,
+                            self.case_run_1.case_text_version)
+        response = self.client.post(self.update_url,
+                                    {'case_run': [self.case_run_1.pk]},
+                                    follow=True)
+
+        self.assertNotContains(response, _('%d CaseRun(s) updated:') % 1)
```

Comment:
FYI this passes b/c when I don't have permissions I am redirected to the login page which doesn't contain this text.